.firebaserc

```diff
@@ -1,6 +1 @@
 {}
-{
```

Comment:
The command `firebase deploy` defaults to `material2-docs-dev`, however, we can control this by adding the instance name as a parameter (refer to `publish-dev`).